phost/src/main.rs

```diff
@@ -832,9 +875,16 @@ async fn bridge(args: Args) -> Result<(), Error> {
         authory_set_state: None,
     };
 
-    let paraid_storage_key = chain_client::get_paraid_key(&paraclient)
-        .await
-        .expect("Failed to get paraid storage key");
+    let para_id = match chain_client::get_paraid(&paraclient)
+        .await?
+    {
+        Some(pid) => pid,
+        None => {
+            return Err(Error::FailedToDecode);
+        }
+    };
```

Comment:
```suggestion
    let para_id = match chain_client::get_paraid(&paraclient).await?.ok_or(Error::EmptyParaId);
```

Let's also add `Error::EmptyParaId` to `Error`